src/ApiGateways/Mobile.Bff.Shopping/aggregator/Services/IBasketService.cs

```diff
@@ -1,13 +1,9 @@
-﻿using Microsoft.eShopOnContainers.Mobile.Shopping.HttpAggregator.Models;
-using System.Threading.Tasks;
+﻿namespace Microsoft.eShopOnContainers.Mobile.Shopping.HttpAggregator.Services;
 
-namespace Microsoft.eShopOnContainers.Mobile.Shopping.HttpAggregator.Services
+public interface IBasketService
 {
-    public interface IBasketService
-    {
-        Task<BasketData> GetById(string id);
+    Task<BasketData> GetById(string id);
```

Comment:
I'm not a fan of the inconsistency here. I would ideally like to see APIs that are consistently suffixed with "Async" for all task-like returning methods. Are we able to change this to:

```suggestion
    Task<BasketData> GetByIdAsync(string id);
```

Reply:
Ok, sure.